packages/pdk-nag/src/packs/aws-prototyping-rules.ts

```diff
@@ -13,25 +13,990 @@ export let PackName = "AwsPrototyping";
+  },
+  {
+    info: "The ElastiCache cluster uses the default endpoint port.",
+    explanation:
+      "Port obfuscation (using a non default endpoint port) adds an additional layer of defense against non-targeted attacks (i.e. Redis port 6379 and Memcached port 11211).",
+    level: NagMessageLevel.WARN,
+    rule: rules.elasticache.ElastiCacheClusterNonDefaultPort,
+  },
+  {
+    info: "The ElastiCache Redis cluster does not have both encryption in transit and at rest enabled.",
+    explanation:
+      "Encryption in transit helps secure communications to the cluster. Encryption at rest helps protect data at rest from unauthorized access.",
+    level: NagMessageLevel.WARN,
+    rule: rules.elasticache.ElastiCacheRedisClusterEncryption,
+  },
+  {
+    info: "The ElastiCache Redis cluster does not use Redis AUTH for user authentication.",
+    explanation:
+      "Redis authentication tokens enable Redis to require a token (password) before allowing clients to execute commands, thereby improving data security.",
+    level: NagMessageLevel.WARN,
+    rule: rules.elasticache.ElastiCacheRedisClusterRedisAuth,
+  },
+  {
+    info: "The Elastic Beanstalk environment does not upload EC2 Instance logs to S3.",
+    explanation:
+      "Beanstalk environment logs should be retained and uploaded to Amazon S3 in order to keep the logging data for future audits, historical purposes or to track and analyze the EB application environment behavior for a long period of time.",
+    level: NagMessageLevel.WARN,
+    rule: rules.elasticbeanstalk.ElasticBeanstalkEC2InstanceLogsToS3,
+  },
+  {
+    info: "The Elastic Beanstalk environment does not have managed updates enabled.",
+    explanation:
+      "Enable managed platform updates for beanstalk environments in order to receive bug fixes, software updates and new features. Managed platform updates perform immutable environment updates.\n\nExample threat: An actor with a network path to the Elastic Beanstalk environment can attempt to take advantage of a known vulnerability in a platform component used by Elastic Beanstalk, which may lead to unknown impacts possibly impacting the confidentiality, integrity and availability of the data and resource assets associated with the prototype",
+    level: NagMessageLevel.ERROR,
+    rule: rules.elasticbeanstalk.ElasticBeanstalkManagedUpdatesEnabled,
+  },
+  {
+    info: "The Elastic Beanstalk environment is not configured to use a specific VPC.",
+    explanation:
+      "Use a non-default in order to separate your environment from default resources.",
+    level: NagMessageLevel.WARN,
+    rule: rules.elasticbeanstalk.ElasticBeanstalkVPCSpecified,
+  },
+  {
+    info: "The ALB does not have invalid HTTP header dropping enabled.",
+    explanation:
+      "Ensure that your Application Load Balancers (ALB) are configured to drop http headers. Because sensitive data can exist, enable encryption in transit to help protect that data",
```

Comment:
Ack, will change to:

> Ensure that your Application Load Balancers (ALB) are configured to drop http headers. Because sensitive data can exist in these headers.